Realm/Tests/tests.py

```diff
@@ -1,58 +1,69 @@
 import os, re
```

Comment:
[q] What is this file?

Reply:
This file generates PrimitiveArrayPropertyTests.m.